### ISSUE TITLE
Client Mute does not work after TS Update to 3.5.2

### ISSUE THREAD

**Issue by glowfishDE** [ZerGo0/streamdeck-teamspeak3integration#3]
Hi,

after the TS update to 3.5.2 (Client Query = 1.5) the mute does not work anymore.

Checked for a new auth token but everything is fine.

Probably a problem in the TS code but i am not a developer.

**Comment by PythonInvasion** [comment]
What exact don't work?
I got Teamspeak Client 3.5.2 and everythink work fine. Notice that "Toggle INPUT (Local) Mute" is an an invisible mute. No mute icon appears. >> Use instead "Toggle INPUT Mute".

Does "Toggle Output Mute" work? If not, create a new API token (in Teamspeak Client), add it to streamdeck/command and try again.

**Comment by glowfishDE** [comment]
Hi! Sorry i did not provide enough info.

I use the streamdeck "Toggle INPUT Mute" Plugin from you. Before i´ve updated my TS3 today, it was muting my microphone and showed the "Microphone muted" icon (and played that sound) in TS3.

Since today, i can press the button on the stream deck as often as i want - nothing happens in TS3.
The icon updates on the Stream Deck but the microphone input stays open in TS3.

I created a new ClientQueryAPI Key just to be sure but it does not help.

I re-checked just right now - and for reasons i dont know - it has a wrong API Key in the settings of TS3 / Streamdeck.

I corrected the API Key and it works like a charm again. Sorry :( I checked that before i made the post but somehow the API key got replaced by another.
